app/common/src/cljc/teet/document/document_model.cljc

```diff
@@ -0,0 +1,38 @@
+(ns teet.document.document-model
+  (:require [clojure.string :as str]))
+
+(def ^:const upload-max-file-size (* 1024 1024 100))
```

Comment:
100 MB seems to me a small limit for some kinds of files (detailed technical PDFs, zip files) ?